google_guest_agent/uefi/uefi.go

```diff
@@ -0,0 +1,57 @@
+// Package uefi provides utility functions to read UEFI variables.
+package uefi
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+const (
+	defaultEFIVarsDir = "/sys/firmware/efi/efivars"
+)
+
+// VariableName represents UEFI variable name and GUID.
+// Format: {VariableName}-{VendorGUID}
+type VariableName struct {
+	Name string
+	GUID string
+}
+
+// Variable represents UEFI Variable and its contents.
+type Variable struct {
+	Name       VariableName
+	Attributes []byte
+	Content    []byte
+}
+
+// VariablePath returns a path for UEFI variable on disk.
+func VariablePath(v VariableName) string {
```

Comment:
What about we transform this `VariablePath()` function into `func (v Variable) String()...` ?

Reply:
Done